modules/apps/portal-language/portal-language-override-test/src/testFunctional/tests/LanguageOverride.testcase

```diff
@@ -390,6 +390,35 @@ definition {
 		}
 	}
 
+	@description = "It asserts proper override and translation infos for new language key."
+	@priority = "4"
+	test CanViewProperOverrideInfosforNewLanguageKey {
+		task ("Given one new language key is created with all language override translations") {
+			LanguageOverride.addLanguageKey(
+				languageKey = "month",
+				translationOverrideList = "en-US,ar-SA,ca-ES,zh-CN,nl-NL,fi-FI,fr-FR,de-DE,hu-HU,ja-JP,pt-BR,es-ES,sv-SE",
+				translationOverrideValueList = "month,دقيقة,mes,月,maand,kuukausi,mois,Monat,hónap,のメモ,mês,mes,månad");
+		}
+
+		task ("When the user filters by 'Any Language' and change language to "pt-BR"") {
+			ManagementBar.setFilterAndOrder(filterBy = "Any Language");
+
+			PageEditor.changeLocale(
+				currentLocale = "en-US",
+				locale = "pt-BR");
+		}
+
+		task ("Then the key should appear and list all languages it has translations for") {
+			LexiconEntry.viewEntryName(rowEntry = "month");
+
+			LexiconEntry.viewEntryName(rowEntry = "mês");
```

Comment:
Use the _LexiconList.viewListEntrySubtitle_ to replace if it works, since _LexiconEntry.viewEntryName_ makes others misunderstand.